bliss/models/galaxy_net.py

```diff
@@ -97,6 +97,7 @@ def __init__(
         self.register_buffer("zero", torch.zeros(1))
         self.register_buffer("one", torch.ones(1))
 
+        # FIXME it's available through self.hparams["optimizer_params"]
```

Comment:
it would be great if you could resolve this FIXME's in this PR :)  (I tend to forget about inline TODO comments)

Reply:
since they are pretty small I wouldn't bother with a separate PR